File: Naloga 36/naloga_36.py
def naloga_36():
    stevilo = int(input("vnesi stevilo"))
    stevila = [i for i in range(10, 0, -1)]
    stevila.reverse()
    while (len(stevila)):
        print(''.join(map(str, stevila)))
        stevila.pop(-1)

if __name__ == '__main__':
    naloga_36()

File: Naloga 36/naloga_36.cpp
#include <iostream>
#include <vector>

void naloga_36 () {
    int stevilo;
    std::cin >> stevilo;
    while (stevilo) {
        for (int i = 1; i<=stevilo;++i) {
            std::cout << i;
        };
        std::cout << std::endl;
        --stevilo;
    }
};

int main () {
    naloga_36();
    return 0;
};
